resident-services/scripts/Resident_Test_script.jmx

```diff
@@ -2861,6 +2861,764 @@ vars.put(&quot;utcTime&quot;, utcTime);</stringProp>
           <hashTree/>
         </hashTree>
       </hashTree>
+      <ThreadGroup guiclass="ThreadGroupGui" testclass="ThreadGroup" testname="Identity Info Schema Type (Execution)" enabled="true">
+        <stringProp name="ThreadGroup.on_sample_error">continue</stringProp>
+        <elementProp name="ThreadGroup.main_controller" elementType="LoopController" guiclass="LoopControlPanel" testclass="LoopController" testname="Loop Controller" enabled="true">
+          <boolProp name="LoopController.continue_forever">false</boolProp>
```

Comment:
Identity Mapping (Execution) - Response Assertion - why error assertion is not done as usual ?

Reply:
Same question for all API endpoints "Response Assertion"

Reply:
Where ever we have used response assertion as **"errors":null** the correct response for the particular APIs consist of **"errors":null** only. So that is why have kept the response assertion like this.